### PR TITLE
Tk update documentation week 13

### PR DESCRIPTION
**Pull request by teekaytai**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -861,6 +863,60 @@ testers are expected to do more *exploratory* testing.
...
+1. Modifying interviews in sorted list
+
+    1. Prerequisites: Sort the applications list by interview (either in forward or reverse order). Multiple different applications in list. Multiple applications with interviews and without interviews.
+
+    1. Test case: Add an interview to an application without an interview.<br>
+       Expected: The application moves from near the bottom of the application list to the correct position such that the list remains sorted in order of interview date.
+
+    1. Test case: Remove an interview from an application with an interview.<br>
+       Expected: The application moves to near the bottom of the application list together with the other applications with no interview.
+
```
Comment (eesung00) [inline docs/DeveloperGuide.md:907]: The numbering is incorrect

**Review by eesung00 — COMMENTED** [review]
LGTM 👍 Just one problem spotted

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (teekaytai) [reply docs/DeveloperGuide.md:907]: Oh don't worry, in markdown the list numbering doesn't affect the numbering that actually gets displayed as long as the first entry is labelled with a 1. The benefit of doing this is not having to edit later numbers when you edit one of the entries
https://www.markdownguide.org/basic-syntax/#ordered-lists

**Review by jhchee18 — COMMENTED** [review]
LGTM 👍

**Pull request merged**